### PR TITLE
hermit add to readme

### PR DESCRIPTION
**Pull request by DaveSys911**
**What does this PR do?**

Adds hermit to the main readme section
**Related Issues**

**Checklist**
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary README/documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,16 +15,29 @@
...
 2. Elasticsearch with the default username & password (`elastic` & `changeme`) running on the default port (`http://localhost:9200`)
 3. Kibana with running on the default port (`http://localhost:5601`)
 4. Install and configure [Elastic-Package](https://github.com/elastic/elastic-package)
 5. Set up the local env:
 
-```zsh
-just setup-env
-```
-
+- Install & activate hermit<br />
```
Comment (olegsu) [inline README.md:24]: replace <br /> with two spaces for new line

Comment (DaveSys911) [reply README.md:24]: Done

**Comment by DaveSys911** [comment]
@olegsu @gurevichdmitry  Could u please review?

**Review by olegsu — APPROVED** [review]

**Review by gurevichdmitry — APPROVED** [review]
@DaveSys911, LGTM

**Pull request merged**